core/src/actors/node.rs

```diff
@@ -1,29 +1,28 @@
 use std::io;
-use std::net::SocketAddr;
 use std::process::exit;
 
 use actix::{Actor, System};
 use log::info;
 
-use crate::actors::client::Client;
 use crate::actors::config_manager::ConfigManager;
+use crate::actors::connections_manager::ConnectionsManager;
 use crate::actors::peers_manager::PeersManager;
-use crate::actors::server::Server;
-use crate::actors::session_manager::SessionManager;
+use crate::actors::sessions_manager::SessionsManager;
 use crate::actors::storage_manager::StorageManager;
 
-/// Function to run the main system
-pub fn run(address: SocketAddr, db_root: &str, callback: fn()) -> io::Result<()> {
-    info!("Witnet server listening on {}", address);
+// Default configuration filename
+const CONFIG_DEFAULT_FILENAME: &str = "witnet.toml";
```

Comment:
I would move this constant to the `ConfigManager` module, and create a helper method:

```rust
ConfigManager::from_default_file()
```